libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackHelper.cs

```diff
@@ -110,5 +114,15 @@ public static NewSlackMessage GetMessageFromSlackEvent(dynamic slackEvent)
                 channel = eventProperty.channel ?? null,
             };
         }
+
+        public static async Task SendHttpResponse(HttpResponse httpResponse, int responseStatusCode, string responseText)
```

Comment:
The CancellationToken is missing here and the method name should be async. Let's fix that in the next refactor.